README.md

```diff
@@ -2,9 +2,118 @@
 
 [![Build Status](https://travis-ci.org/deliveroo/crypto_env_var.svg?branch=master)](https://travis-ci.org/deliveroo/crypto_env_var)
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/crypto_env_var`. To experiment with that code, run `bin/console` for an interactive prompt.
+**CryptoEnvVar** provides:
+
+* [Asymmetric encryption](#asymmetric-encryption) of JSON-serializable Ruby objects using a RSA keypair.
+* [Secrets management for the Ruby ENV](#secrets-management-for-the-ruby-env).
+
+## Asymmetric Encryption
+
+The gem provides two functions that can be used on their own, even without the ENV management functionality.
+
+Any Ruby object that can be serialized and deserialized with `JSON.dump` nd `JSON.load` is a valid input. This means that you can use a Hash, but that symbols won't be preserved. The output is a base64-encoded string, safe to be stored and transferred as text.
+
+```ruby
+rsa_private_key_string = File.read("path/to/rsa_key.pem")
+rsa_public_key_string  = File.read("path/to/rsa_key.pub.pem")
+
+data = { "foo" => 42, "bar" => [1, 3, 3, 7], "baz" => true }
+
+encrypted_string = CryptoEnvVar.encrypt(data, rsa_private_key_string)
```

Comment:
Nit: usually in crypto examples one sees `plaintext` and `ciphertext` being used for variable names.

Reply:
Ops. Nice catch. I think I've used plaintext and ciphertext elsewhere but not here.